### PR TITLE
Use correct style for re-exported imports (for 3.5 branch)

### PR DESCRIPTION
**Pull request by AMDmi3**
## What do these changes do?

Use correct style for re-exported imports, makes `mypy --strict` happy.

## Are there changes in behavior for the user?

No

## Related issue number

#3868

## Checklist

- [x] I think the code is well written
- [ ] Unit tests for the changes exist (no new code and functional changes though, so should be covered by existing tests)
- [ ] Documentation reflects the changes
- [x] If you provide code modification, please add yourself to `CONTRIBUTORS.txt`
- [x] Add a new news fragment into the `CHANGES` folder

### CODE REVIEW

**Comment by AMDmi3** [comment]
Please give me some more time, I was not aware `from foo import (bar as baz)` syntax is possible. Will redo the patches that way, that will also fix flake8.

File: aiohttp/web_ws.py [hunk 1]
```diff
@@ -12,18 +12,16 @@
...
-    WS_KEY,
-    WebSocketError,
-    WebSocketReader,
-    WebSocketWriter,
-    WSMessage,
-    WSMsgType,
-    ws_ext_gen,
-    ws_ext_parse,
-)
+from .http import WS_CLOSED_MESSAGE
```
Comment (webknjaz) [inline aiohttp/web_ws.py]: This change is totally unrelated

**Comment by asvetlov** [comment]
Please fix merge conflicts

**Comment by AMDmi3** [comment]
Updated
